bot/exts/moderation/infraction/management.py

```diff
@@ -167,8 +163,8 @@ async def infraction_edit(
                 self.infractions_cog.schedule_expiration(new_infraction)
 
             log_text += f"""
-                Previous expiry: {until_expiration(infraction['expires_at']) or "Permanent"}
-                New expiry: {until_expiration(new_infraction['expires_at']) or "Permanent"}
+                Previous expiry: {time.until_expiration(infraction['expires_at'])}
```

Comment:
Similar as the above, this fails with the below error when trying to edit an infraction without an expiry.
```
bot_1       | Traceback (most recent call last):
bot_1       |   File "/usr/local/lib/python3.9/site-packages/discord/ext/commands/core.py", line 85, in wrapped
bot_1       |     ret = await coro(*args, **kwargs)
bot_1       |   File "/bot/bot/exts/moderation/infraction/management.py", line 134, in infraction_edit
bot_1       |     expiry = time.format_with_duration(duration)
bot_1       |   File "/bot/bot/utils/time.py", line 303, in format_with_duration
bot_1       |     duration = humanize_delta(timestamp, other_timestamp, max_units=max_units)
bot_1       |   File "/bot/bot/utils/time.py", line 196, in humanize_delta
bot_1       |     start = arrow.get(args[1]) if len(args) == 2 else arrow.utcnow()
bot_1       |   File "/usr/local/lib/python3.9/site-packages/arrow/api.py", line 80, in get
bot_1       |     return _factory.get(*args, **kwargs)
bot_1       |   File "/usr/local/lib/python3.9/site-packages/arrow/factory.py", line 228, in get
bot_1       |     raise TypeError("Cannot parse argument of type None.")
bot_1       | TypeError: Cannot parse argument of type None
```

Reply:
Sorry, this is comment is on the wrong line. This error is actually for line 123 in here.